### PR TITLE
browserpass: init at 1.0.2

### PR DESCRIPTION
**Pull request by pajowu**
###### Motivation for this change

The browser-extension for pass (browserpass) needs a native component. That component is installed using this package. The Config-files are added to chrome and chromium with the `programs.browserpass` module.

###### Things done

- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [x] macOS
   - [x] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by danielfullmer** [comment]
I have also been using this very useful chrome extension in my own nixos configuration, but hadn't packaged it up as a module like in this pull request. If it's useful, see these as well:
https://github.com/danielfullmer/nixrc/blob/master/pkgs/browserpass/default.nix
and
https://github.com/danielfullmer/nixrc/blob/master/profiles/desktop/default.nix#L197

@pajowu  Instead of using the prebuilt binary from upstream, I'd recommend using `buildGoPackage` against the source code.

**Comment by pajowu** [comment]
@danielfullmer that looks great, i'll try to change it to build from git. That way it would also support 32bit

**Comment by jagajaga** [comment]
Why not just https://github.com/nwallace/passff? :)

File: pkgs/tools/security/browserpass/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.0
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "browserpass-unstable-${version}";
+  version = "HEAD";
```
Comment (jagajaga) [inline pkgs/tools/security/browserpass/default.nix]: Use proper way to name pkg. In this case it should be `browserpass-(date of commit)`. Also please use `sha` for consistency.

Comment (pajowu) [inline pkgs/tools/security/browserpass/default.nix]: fixed with 11913f2ce1

**Comment by pajowu** [comment]
@jagajaga 

> Why not just https://github.com/nwallace/passff? :)

passff also needs a native extension, which would need a similar install method. Also giving more options seems like a good thing

**Comment by wucke13** [comment]
@ttuegel @pajowu 
This is still not resolved, browserpass does not work for me on the most recent firefox package in NixOS 17.09

**Pull request merged**